lib/seattleflu/id3c/cli/command/etl/redcap_det_uw_reopening.py

```diff
@@ -689,10 +681,16 @@ def create_computed_questionnaire_response(record: REDCapRecord, patient_referen
     """
     # A birthdate of None will return a falsy relativedelta() object
     delta = relativedelta(encounter_date, birthdate)
+
     if not delta:
         age = None
     else:
-        age = delta.years
+        # Age Ceiling
+        try:
+            age = age_ceiling(delta.years)
+        except ValueError:
+            age = None
```

Comment:
Is the try/except needed now that we apply `age_ceiling` directly to the `delta.years`? It was previously added to catch the int conversion error if `core_age_years` was an empty string.

Reply:
Good point! That try/except isn't needed anymore, so I've removed it in the latest commit.
If the incoming birthdate is None then relativedelta will return None, and we'll not set an age.
If the incoming birthdate is not a date then relativedelta will throw a TypeError.